src/coreclr/jit/CMakeLists.txt

```diff
@@ -251,6 +251,10 @@ if (CLR_CMAKE_TARGET_WIN32)
     vartype.h
   )
 
+  # Append clrjit.natvis file
+  list (APPEND JIT_SOURCES
+    clrjit.natvis)
```

Comment:
Is this is redundant?

Reply:
no, we still want it so ninja can rebuild when it changes.

Reply:
it's a no op on platforms other than windows, you can move it under the win32 block you created